components/Homepage/RoundActionCard/RoundActionCard.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  Center,
+  HStack,
+  Heading,
+  Spinner,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import { Phase } from "services/PhaseMgmtService";
+
+interface Props {
+  phase: Phase;
```

Comment:
nit - let's put this is `roundInfo` object of `{phase: Phase, roundId: number}`, since they'll always move together